.github/workflows/cli.yaml

```diff
@@ -42,6 +46,20 @@ jobs:
           then
             echo "TEST_GIT_BRANCH=${{ github.event.pull_request.base.ref }}" >> $GITHUB_ENV
           fi
+      - name: Install helm
+        id: helm
+        uses: azure/setup-helm@5119fcb9089d432beecbf79bb2c7915207344b78 # v3.5
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Install chainsaw
+        uses: kyverno/action-install-chainsaw@d12e54dd3506268b9d5bb61fbbcb2e7eec5fb70c # v0.1.1
+      # create cluster
+      - name: Create kind cluster
+        uses: helm/kind-action@dda0770415bac9fc20092cacbc54aa298604d140 # v1.8.0
+        with:
+          node_image: kindest/node:v1.29.0
+          cluster_name: kind
+          config: ./scripts/config/kind/default.yaml
```

Comment:
We don't need to create a KinD cluster here. There is a flag to Chainsaw called `--noCluster` which enables Chainsaw to run without looking for a connection to a cluster. cc @eddycharly

Reply:
Can you tell me How to run the tests without cluster?l

Reply:
Pass the flag I gave you to chainsaw.

Reply:
I tried with chainsaw test --no-cluster but it errors as unknown flag --no-cluster

Reply:
@eddycharly

Reply:
Is I need to use the different version of chainsaw?

Reply:
Yes, you need to install at least 0.1.3. 0.1.4 is preferable.

Reply:
Can you please tell me how to checkout the latest version of chainsaw on github actions?